main.py

```diff
@@ -1,10 +1,66 @@
+import json
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
+def load_players_data() -> dict:
+    with open("players.json", "r") as file:
+        return json.load(file)
+
+
+def get_or_create_race(race_data: dict) -> Race:
+    return Race.objects.get_or_create(
+        name=race_data["name"],
+        defaults={"description": race_data.get("description", "")}
+    )[0]
+
+
+def create_skills_for_race(race: Race, skills_data: list) -> None:
+    for skill_data in skills_data:
+        Skill.objects.get_or_create(
+            name=skill_data["name"],
+            defaults={"bonus": skill_data["bonus"], "race": race}
```

Comment:
Ensure that the `bonus` field is correctly handled if it is intended to be a numerical value. This might require changes in the `create_skills_for_race` function.